README.md

```diff
@@ -1,102 +1,114 @@
-Setting up for local development
-================================
+# Setting up for local development
 
-To work on hosted rapidpro:
+## 1. Create an ngrok account
 
+Use ngrok to set up a URL that fowards to your development server. You'll need
+a paid account so that you can set up a subdomain.
+
+1. Download the ngrok zip: https://ngrok.com/download
+
+2. Unzip the package, and move the `ngrok` executable to a convenient
+   location.
+
+3. Log into https://ngrok.com using the Google login with your Caktus account.
+   This will give you access to the "Caktus Group" organization, which is
+   a paid account.
+
+4. Configure ngrok to use the Caktus account.
+   Get the auth token here from https://dashboard.ngrok.com/auth
+
+    ```
+    ./ngrok authtoken <caktus-auth-token>
+    ```
+
+   NOTE: You probably have access to an individual account as well as the
+   Caktus organization account. Make sure that "Caktus Group" is selected
+   from the account dropdown. Otherwise, you'll see this error when you start
+   ngrok later: "Only paid plans may bind custom subdomains"
+
+5. Decide upon a unique subdomain for your use in development, e.g.
+   MYSUBDOMAIN.ngrok.io. No configuration is needed yet, but you'll use
+   this subdomain/url throughout the rest of the setup.
+
+## 2. Create a Twilio phone number
+
+1. Get access to a Twilio account. Ask for access to the Caktus account
+   credentials on Lastpass.
+
+2. Create a new Twilio phone number for your use in development:
+   https://www.twilio.com/user/account/phone-numbers/incoming
+
+   NOTE: This adds a small cost to our account; that's allowed.
+
+3. Configure your new number with "TwiML App", and click "+ Create a new TwiML app",
```

Comment:
I don't think this step is necessary. RapidPro did this for me when I added my Twilio channel.  (But I had to have put the right subdomain into my Django settings; the first time I added the channel, I hadn't yet done that, and RapidPro set it up using the wrong subdomain.)